common/plugins/eu.esdihumboldt.hale.common.core/src/eu/esdihumboldt/hale/common/core/io/impl/GZipEnabledImport.java

```diff
@@ -64,6 +65,15 @@ public URI getUsedLocation() {
 			return getLocation();
 		}
 
+		/**
+		 * @see eu.esdihumboldt.hale.common.core.io.supplier.LocatableInputSupplier#getUsedLocations()
+		 */
+		@Override
+		public List<URI> getUsedLocations() {
```

Comment:
From the usage and the number of implementations that just return null in the implementation I think the `LocatableInputSupplier` interface is not the right location to add this method.
Maybe instead only in `FileIOSupplier`? Or even a new supplier class that is dedicated to be a source for multiple files?

Reply:
Created a new class `FilesIOSupplier` or should I rename it to `MultipleFilesIOSupplier`?

Reply:
It's perfect, thank you!